### PR TITLE
feat: ✨ LS_COLORS generated with vivid

### PR DESCRIPTION
**Pull request by tkapias**
I generated a theme for LS_COLORS with vivid as suggested in #56. 

Use the theme yml file with vivid or export the variable included in the .bashrc file in your shell.

### CODE REVIEW

**Comment by tkapias** [comment]
image
image

**Review by fenetikm — CHANGES_REQUESTED** [review]
Thanks - just one little thing I think.

**Comment by tkapias** [comment]
I changed the name of the file and updated the colors, it's much better.

I did not put a comment, but LS_COLORS is useful for many commands like ls, tree, find, du, lsd... I let you decide if it needs to be commented.

image
image
image

File: vivid/.bashrc [hunk 1]
```diff
@@ -0,0 +1,2 @@
...*INSTALL.txt=1;38;2;180;180;185:*LICENSE-MIT=1;38;2;180;180;185:*MANIFEST.in=0;38;2;153;164;188:*Makefile.am=0;38;2;153;164;188:*Makefile.in=0;38;2;54;54;58:*.applescript=0;38;2;99;81;150:*.fdb_latexmk=0;38;2;54;54;58:*.webmanifest=0;38;2;255;118;26:*CHANGELOG.md=1;38;2;180;180;185:*CONTRIBUTING=1;38;2;180;180;185:*CONTRIBUTORS=1;38;2;180;180;185:*appveyor.yml=0;38;2;153;164;188:*configure.ac=0;38;2;153;164;188:*.bash_profile=0;38;2;99;81;150:*.clang-format=0;38;2;153;164;188:*.editorconfig=0;38;2;153;164;188:*CHANGELOG.txt=1;38;2;180;180;185:*.gitattributes=0;38;2;153;164;188:*.gitlab-ci.yml=0;38;2;153;164;188:*CMakeCache.txt=0;38;2;54;54;58:*CMakeLists.txt=0;38;2;153;164;188:*LICENSE-APACHE=1;38;2;180;180;185:*pyproject.toml=0;38;2;153;164;188:*CODE_OF_CONDUCT=1;38;2;180;180;185:*CONTRIBUTING.md=1;38;2;180;180;185:*CONTRIBUTORS.md=1;38;2;180;180;185:*.sconsign.dblite=0;38;2;54;54;58:*CONTRIBUTING.txt=1;38;2;180;180;185:*CONTRIBUTORS.txt=1;38;2;180;180;185:*requirements.txt=0;38;2;153;164;188:*package-lock.json=0;38;2;54;54;58:*CODE_OF_CONDUCT.md=1;38;2;180;180;185:*.CFUserTextEncoding=0;38;2;54;54;58:*CODE_OF_CONDUCT.txt=1;38;2;180;180;185:*azure-pipelines.yml=0;38;2;153;164;188'
```
[the remark this answers is not in the document]
Comment (tkapias) [reply vivid/.bashrc]: Done

**Comment by fenetikm** [comment]
LGTM, cheers 🍻

**Pull request merged**